config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -24,9 +22,8 @@ periodics:
         - --test=false
         - --check-leaked-resources
         - --check-version-skew=false
-        # TODO: change to release/stable-x.y
-        - --extract=ci/latest
-        - --extract=ci/latest-1.19
+        - --extract=ci/stable
+        - --extract=ci/stable-1.19
```

Comment:
I'm fuzzy about what these represent... is ci/stable the latest 1.20.x release or the latest commit on release-1.20 branch? Same question for ci/stable-1.19.

If we want to test upgrade from the latest release to current dev branch, should this be `ci/latest` and `ci/stable`?

Reply:
ci/latest | ci/stable 
-|-
v1.21.0-alpha.0 | v1.20.1

ci/latest-1.20 | ci/stable-1.20
-|-
v1.20.2-rc.0 | v1.20.1

yea, ci/stable is the latest 1.20.x while ci/latest has the largest tag number. e.g. v1.21.0-alpha.0

with bound beta, latest or stable will eventually have bound enabled by default and this upgrade will not have much value.

my current plan is fix the initial version to versions with bound alpha (e.g. ci/stable-1.19) and upgrade to ci/stable.

Reply:
unless we want this job to tell us something broke **after** we cut a release, I'm pretty sure the version we're upgrading to should be using **latest** instead of **stable**

given we also didn't enable BoundServiceAccountTokenVolume by default in 1.20, I think this should probably just test from `ci/latest-1.20` to `ci/latest` for now.

Reply:
sounds good.